daemon/stop.go

```diff
@@ -82,7 +92,6 @@ func (daemon *Daemon) containerStop(container *containerpkg.Container, seconds i
 			logrus.Warn(err) // Don't return error because we only care that container is stopped, not what function stopped it
 		}
 	}
-
```

Comment:
Please clean up your patch to not introduce extras like this.

Reply:
ok，check it

Reply:
ok，I will fix it